src/lib.rs

```diff
@@ -81,20 +88,52 @@ mod collections;
 mod config;
 mod console;
 mod drivers;
-mod environment;
+pub mod environment;
```

Comment:
This needs to be public, so the test can check if it is running on uhyve when exiting. Otherwise it assumes it's on qemu and attempts to exit via the debug device. If that fails, the normal `sys_exit` is used.